libs/dev-dock/frontend/src/dev_dock.tsx

```diff
@@ -660,9 +674,14 @@ function DevDock() {
           </Column>
           <Column>
             <ScreenshotControls containerRef={containerRef} />
-            {APPS_WITH_PRINTER.includes(machineType) && <PrinterMockControl />}
+            {printerType === 'standard' && <PrinterMockControl />}
           </Column>
         </Row>
+        {printerType === 'fujitsu' && (
+          <Row>
+            <FujitsuPrinterMockControl />
+          </Row>
+        )}
```

Comment:
It made more sense to break this out in a row because if it's in a column it takes up an obnoxious amount of horizontal space with the dropdown values.